### DISCUSSION TITLE
Primevue Editor bullet list option in toolbar adds ordered list

### DISCUSSION THREAD

**Discussion by AlexBindevald** [primefaces/community#1841] [PrimeVue]
Hello,

I have a problem when using Primevue Editor in my very basic Vue application. Without any modifications to Editor component, the bullet list option in toolbar creates ordered list instead of unordered list. In primevue homepage and stackblitz it works fine. What could be the issue?

Steps I have done so far:
Created new Vue 3 app, installed primevue and quill according to instructions

**Comment by cortvi** [comment]
Same thing here

**Comment by netkreacije** [comment]
Same here, did anyone maybe found a solution for this issue?

**Comment by asakahn** [comment]
**More information here:**
https://github.com/slab/quill/issues/2614#issuecomment-558789994

**Comment by asakahn** [comment]
The new Quill Editor v2 uses `<ol>` tags even for unordered (bullet) lists and uses the `data-list='bullet'` attribute on the `<li>` to determine whether it's an ordered or bullet list. More info: https://github.com/slab/quill/issues/3957#issuecomment-1895961689

Currently, PrimeVue's Editor component style file `node_modules/primevue/editor/style/index.mjs` has this code, which is automatically adding the numbered list to all `<li>` tags:

`.ql-editor ol li:before {\n    content: counter(list-0, decimal) '. ';\n}`

Another line of CSS should be added like so to make it more specific to the attribute type:

`.ql-editor ol li[data-list="bullet"]::before {
    content: '\2022';
}`


### A temporary fix for people coming across issue is this:

**Editor.vue**
```
<script setup lang="ts">
import Editor from 'primevue/editor';
</script>

<template>
<Editor />
</template>

<style>
.ql-editor ol li[data-list="bullet"]::before {
    content: '\2022';
}
</style>
```

**Reply by Andy-Symphony-App** [reply]
This only seems to work in the preview window for me. As soon as I save it, the bullet points disappear.  So I used this instead:
`.ql-ui::before {
    content: '\2022';
    /* Unicode for bullet point */
    margin-right: 0.5em;
    /* Add space after the bullet */
    color: black;
    /* Set the bullet color if needed */
}`

**Comment by RoyvanEmpel** [comment]
Primevue should be using the getSementicHTML() for the exporting of the text value. Without this I won't be able to use the Primevue version and need to implement the Quill editor myself. I'm trying to use the editor as a E-mail WYSIWYG editor, now i need to add some wierd css to make sure that the ordered list is rendered as a unordered list... I don't think that this will work in every e-mail client.

[Reference to the docs:](https://quilljs.com/docs/api#getsemantichtml)
> **getSemanticHTML**
> Get the HTML representation of the editor contents. This method is useful for exporting the contents of the editor in a format that can be used in other applications.

Is there another way of calling this function with the primevue version?
